region/maridia/outer/Mama Turtle Room.json

```diff
@@ -760,6 +793,20 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "link": [4, 6],
+      "name": "Base",
+      "requires": [
+        {"or": [
+          "canWalljump",
+          "SpaceJump",
+          {"and": [
+            "HiJump",
+            "canCarefulJump"
+          ]}
```

Comment:
good call, adding grapple here

Reply:
For the [4, 2] strat, we have a hijump or precise grapple requirement. This is because the block is off screen and not easy to find, since it's a bit to the right of where mama turtle is

Reply:
👍 copying that, result looks like:
```json
    {
      "link": [4, 6],
      "name": "Base",
      "requires": [
        {"or": [
          "canWalljump",
          "SpaceJump",
          {"and": [
            "HiJump",
            "canCarefulJump"
          ]},
          {"and": [
            "canUseGrapple",
            {"or": [
              "HiJump",
              "canPreciseGrapple"
            ]}
          ]}
        ]}
      ]
    }
```

Reply:
(btw, this is the exact kind of thing that made me think splitting above/below grapple into separate nodes would be beneficial... we'd be able to model 4->X in a single spot, then X->2 vs. X->6 become simple)